.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
```

Comment:
Looks like a local config file got included in the commit. I'll handle removing and ignoring after merge.

Reply:
That was me. Looks like the cSpell config file gets saved into the current workspace instead of a global VSC area. I'll avoid using it for now.

Reply:
I'll add a universal ignore for `settings.json` files, so you'll be free to use them going forward.

Reply:
Thanks! Having a local dictionary of terms accessible through VSC is a big help.